images/builder-nodejs/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM docker.io/library/node:20-alpine3.17
+
+SHELL ["/bin/ash","-c"]
+WORKDIR /usr/work
+LABEL org.opencontainers.image.source="https://github.com/dyrector-io/dyrectorio/tree/develop/images/builder-nodejs"
+
+RUN apk add alpine-sdk docker git zstd openssl1.1-compat jq
```

Comment:
Sorry for the stupid question but why need git for these images?

Reply:
@nandor-magyar using them with golang/Makefile iirc. When I asked about it he stated he want to be able to build a package locally too. For other images seemingly not important.

Reply:
If it is only used for golang? this is the builder-nodejs image.